public/app/features/alerting/unified/components/Label.tsx

```diff
@@ -58,8 +58,18 @@ const getStyles = (theme: GrafanaTheme2, color?: string, size?: string) => {
     ? tinycolor2.mostReadable(backgroundColor, ['#000', '#fff']).toString()
     : theme.colors.text.primary;
 
-  const padding =
-    size === 'md' ? `${theme.spacing(0.33)} ${theme.spacing(1)}` : `${theme.spacing(0.2)} ${theme.spacing(0.6)}`;
+  let padding: CSSProperties['padding'] = `${theme.spacing(0.33)} ${theme.spacing(1)}`;
```

Comment:
I think you can do `theme.spacing(0.33, 1)`?

Reply:
That does read more fluent! ✅